src/language-js/print/function-parameters.js

```diff
@@ -261,8 +261,12 @@ function isDecoratedFunction(path) {
       }
       return false;
     },
-    (node, name) => node.type === "VariableDeclarator" && name === "init",
-    (node) => node.kind === "const" && node.declarations.length === 1
+    (node, name) =>
+      (node.type === "VariableDeclarator" && name === "init") ||
+      (node.type === "ExportDefaultDeclaration" && name === "declaration"),
```

Comment:
Same as `export default`, it is better to support `module.export =` and `export =`?